cmds/core/ip/parsing_linux_test.go

```diff
@@ -168,6 +168,75 @@ func TestParseIPNet(t *testing.T) {
 	}
 }
 
+func TestParseAddressorCIDR(t *testing.T) {
+	tests := []struct {
+		name      string
+		cmd       cmd
+		wantIP    net.IP
+		wantIPNet *net.IPNet
+		wantErr   bool
+	}{
+		{
+			name:      "Valid IPv4",
+			cmd:       cmd{Args: []string{"cmd", "192.168.1.1"}},
+			wantIP:    net.ParseIP("192.168.1.1"),
+			wantIPNet: nil,
+			wantErr:   false,
+		},
+		{
+			name:      "Valid IPv6",
+			cmd:       cmd{Args: []string{"cmd", "fe80::1"}},
+			wantIP:    net.ParseIP("fe80::1"),
+			wantIPNet: nil,
+			wantErr:   false,
+		},
+		{
+			name:      "Valid IPv4 CIDR",
+			cmd:       cmd{Args: []string{"cmd", "192.168.1.0/24"}},
+			wantIP:    net.ParseIP("192.168.1.0"),
+			wantIPNet: mustParseCIDR("192.168.1.0/24"),
+			wantErr:   false,
+		},
+		{
+			name:      "Valid IPv6 CIDR",
+			cmd:       cmd{Args: []string{"cmd", "fe80::/64"}},
+			wantIP:    net.ParseIP("fe80::"),
+			wantIPNet: mustParseCIDR("fe80::/64"),
+			wantErr:   false,
+		},
+		{
+			name:      "Invalid address",
+			cmd:       cmd{Args: []string{"cmd", "invalid"}},
+			wantIP:    nil,
+			wantIPNet: nil,
+			wantErr:   true,
+		},
+		{
+			name:      "Invalid CIDR",
```

Comment:
Maybe, as above, we should do `"Invalid IPv4 CIDR"` and `"Invalid IPv6 CIDR"`?

Reply:
Good idea, added in new commit.